tests/drivers/uart/uart_async_api/boards/mimxrt1060_evkb.overlay

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/ {
+       chosen {
+               zephyr,sram = &dtcm;
+       };
+};
+
+&lpuart3 {
+	status = "okay";
+	current-speed = <115200>;
```

Comment:
Not blocking/strictly required, but if you add the `nxp,loopback` property here the test can be run without physically connecting the TX/RX of the LPUART pins